jdaviz/core/astrowidgets_api.py

```diff
@@ -440,10 +441,19 @@ def add_markers(self, table, x_colname='x', y_colname='y',
                 raise AttributeError(f'{getattr(image, "label", None)} does not have a valid WCS')
             sky = table[skycoord_colname]
             t_glue = Data(marker_name, ra=sky.ra.deg, dec=sky.dec.deg)
+            dcomps = image.components
+            if (isinstance(image.coords, gwcs.WCS) and
+                    image.coords.output_frame.reference_frame.name == 'icrs' and

```

Comment:
@larrybradley , I added one extra check here. Is this what you meant?

Reply:
except perhaps it should be `!= 'galactic'` since there are several RA/Dec coordinate frames.

Reply:
Can all these other coordinate frames also give Lon/Lat component names? I still don't understand why this is even a thing in the first place.

Reply:
I think gwcs *always* gives Lon/Lat component names.  @nden

Reply:
OK added a commit with your suggestion. Thanks!

Reply:
If I were to actually change the GWCS object, it would happen in the parser.

Also, redefine them how exactly? Thanks!

Reply:
The default names are `lon`, `lat`  but a project may decide and set them to what makes sense for them, e.g `coordinate_frames.CelestialFrame(reference_frame=coordinates.ICRS(), axes_names=('ra', 'dec'))`
A user cannot redefine them, this is at the level of "designing" the WCS.

Reply:
Oh... thanks for the clarification. In that case, that is out of Jdaviz's control. This WCS is from JWST MIRI data that @orifox and MAST provided.

Reply:
But isn't `jdaviz` supposed to work with any input images?

Reply:
Currently only JWST images have GWCS. We are not working on Roman (yet) and have no plans to support DKIST. This patch only affects GWCS, so I am not sure what your question is referring to. Can you please clarify, @larrybradley ?

Reply:
So, should we move ahead with this PR as-is? Roman support would need new code anyway, since it is pure ASDF, so we don't have to future-proof that much over here.

Reply:
Sure